src/Coriolis/beta_plane.jl

```diff
@@ -12,11 +14,9 @@ end
     BetaPlane([T=Float64;] f₀=nothing, β=nothing,
                            rotation_rate=Ω_Earth, latitude=nothing, radius=R_Earth)
 
-A parameter object for meridionally increasing Coriolis parameter (`f = f₀ + βy`).
-
 The user may specify both `f₀` and `β`, or the three parameters `rotation_rate`,
 `latitude`, and `radius` that specify the rotation rate and radius of a planet, and
-the central latitude at which the `β`-plane approximation is to be made.
+the central latitude (where y = 0) at which the `β`-plane approximation is to be made.
 
 By default, the `rotation_rate` and planet `radius` is assumed to be Earth's.
```

Comment:
This is a tiny comment and not related to this PR --- but its incorrect and might be misleading to say that we "assume" something here. We don't assume anything --- we just have a default. This is only important to avoid confusion; if we say we "assume" something, it could imply that it can't be changed. By allowing the rotation rate and radius to be _set_, we avoid assuming anything (every parameter is adjustable). It's super important that we assume as little as possible in the code.

Reply:
I guess the `y = 0` point is made obvious by the `f = f₀ + βy` equation.

I think it's good to mention though because you can change the y-domain in the grid constructor, so you get a different beta plane with `y = (0, Ly)` and with `y = (-Ly/2, Ly/2)`.

So you're right, we don't assume anything. But maybe it'll be useful to mention that the y-domain can be changed in the grid constructor?

Reply:
Hmm I think I'm just advocating to be careful about language. `f = f₀ + βy` seems like a *definition*, rather than an assumption. I also feel that a "default" is not an assumption.

Reply:
I do agree that it'd good to mention that `y=0` is special. It would not be hard to add a parameter `y₀` either!

Reply:
To clarify, this comment pertains to this line:

> By default, the `rotation_rate` and planet `radius` is assumed to be Earth's.

I advocate we change this to something like

> Earth's `rotation_rate` and planet `radius` are specified by default.